docker-storage-setup.sh

```diff
@@ -637,18 +638,17 @@ scan_disks() {
 }
 
 create_partition() {
-  local dev="$1" size
+  local dev="$1" part="$2" size
 
   # Use a single partition of a whole device
   # TODO:
   #   * Consider gpt, or unpartitioned volumes
   #   * Error handling when partition(s) already exist
-  #   * Deal with loop/nbd device names. See growpart code
   size=$(( $( awk "\$4 ~ /"$( basename $dev )"/ { print \$3 }" /proc/partitions ) * 2 - 2048 ))
     cat <<EOF | sfdisk $dev
 unit: sectors
 
-${dev}1 : start=     2048, size=  ${size}, Id=8e
+${part} : start=     2048, size=  ${size}, Id=8e
```

Comment:
Does this option work for dos table? IOW, who decides what's will be the name of the partition?